src/commonMain/kotlin/exchange.dydx.abacus/processor/wallet/account/SubaccountProcessor.kt

```diff
@@ -238,15 +277,31 @@ internal open class SubaccountProcessor(parser: ParserProtocol) : BaseProcessor(
         return Pair(existing, false)
     }
 
-    private fun receivedFills(
+    private fun processFills(
+        subaccount: InternalSubaccountState,
+        payload: List<IndexerFillResponseObject>?,
```

Comment:
can we make this non-nullable and handle nullability at call site?

Reply:
Can update in followup PR